src/main/webapp/iframe.js

```diff
@@ -102,10 +106,44 @@ function onYouTubeIframeAPIReady() { // eslint-disable-line no-unused-vars
       'onReady': onPlayerReady,
       'onStateChange': onPlayerStateChange,
       'onPlaybackRateChange': onPlayerPlaybackRateChange,
+      'onError': onPlayerError,
     },
   });
 }
 
+let errorMessage;
+function onPlayerError(event) {
+  resetPlaybackInfo();
+  errorTag.style.display = 'block';
+  skipVoteTag.style.display = 'none';
+  const error = event.data;
+  switch (error) {
+    case 2:
```

Comment:
add a comment next to case 2 to explain the error type

Reply:
Sure thing